services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=CinemaHall.objects.get(
+                                    id=cinema_hall_id),
+                                movie=Movie.objects.get(id=movie_id)
+                                )
+
+def get_movies_sessions(session_date: str = None
+                        ) -> QuerySet:
+    date = datetime.strptime(session_date, "%Y-%m-%d")
+    if session_date:
+        return MovieSession.objects.filter(show_time=date)
```

Comment:
According to the checklist, you should use `.get()` when you need to get one item. However, in this case, `.filter()` is appropriate because you are potentially retrieving multiple sessions that match the date.